pkg/operator2/operator.go

```diff
@@ -239,6 +244,19 @@ func (c *authOperator) handleSync(ctx context.Context, operatorConfig *operatorv
 	// will cause a redeploy anyway
 	// TODO move this hash from deployment meta to operatorConfig.status.generations.[...].hash
 	resourceVersions := []string{}
+	// ensure operand namespace and resources exist
+	directResourceResults := resourceapply.ApplyDirectly(c.kubeClientHolder, c.recorder, assets.Asset,
+		"oauth-openshift/ns.yaml",
+		"oauth-openshift/cabundle.yaml",
```

Comment:
nit: remove the `release.openshift.io/create-only: "true"` from the file, it's no longer needed when you do this

Reply:
done

Reply:
done